Changes

```diff
@@ -427,6 +427,11 @@ Working version
   in preparation for quadratic-time fix
   (Gabriel Scherer, review by Enguerrand Decorne)
 
+- #11997: translate structured constants into their Obj.t representation
+  at compile time rather than run time. Changes the way dumpobj prints
```

Comment:
I disagree with the "rather than run time" part.  Currently, the translation is performed at link-time: static link-time in the bytecode linker, run-time link-time in Dynlink.  Suggested change:
```suggestion
- #11997: translate structured constants into their Obj.t representation
  at compile time rather than link time. Changes the way dumpobj prints
```